src/components/stepWizard/step-wizard.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import { Steps } from "./Steps";
+import { stepArray } from "./example usage/page";
+import Step from "./Step";
+
+export const stepWizard: React.FC = () => {
+  return (
+    <Steps totalSteps={3}>

```

Comment:
`totalSteps` assignment can be improved by using `stepArray.length` to keep it dynamic in the story.